stumpy/core.py

```diff
@@ -2608,6 +2608,12 @@ def _merge_topk_PI(PA, PB, IA, IB):
     -------
     None
     """
+    if PA.ndim == 1:
+        mask = PB < PA
+        PA[mask] = PB[mask]
+        IA[mask] = IB[mask]
+        return
+
```

Comment:
Is this okay? I thought this is a small change for case `k=1` and may not need to have its own function. (I did the same thing for `_merge_topk_ρI` and their naive versions)

Reply:
I think instead of `return`, I'd just add `else` and indent everything below for the 2D case

Reply:
> I'd just add `else` and indent everything below for the 2D case

I just remembered you mentioned once that you prefer `if-else` as it is a better practice, and you also prefer not to  to use `return` in the middle of function. I will revise it. Thanks for the feedback!!!